modules/social_features/social_event/social_event.install

```diff
@@ -1800,66 +1800,83 @@ function social_event_update_8915() {
 }
 
 /**
- * Add the 'enable event enrolment' field.
- */
-function social_event_update_8916() {
-  $config_path = drupal_get_path('module', 'social_event') . '/config/static';
-  $source = new FileStorage($config_path);
-  $entity_type_manager = \Drupal::entityTypeManager();
-
-  /** @var \Drupal\Core\Config\Entity\ConfigEntityStorageInterface $field_storage_config_storage */
-  $field_storage_config_storage = $entity_type_manager->getStorage('field_storage_config');
-
-  // Create field storages.
-  $field_type_manager = \Drupal::service('plugin.manager.field.field_type');
-  $data = $source->read('field.storage.node.field_event_enable_enrollment_8916');
-  $class = $field_type_manager->getPluginClass($data['type']);
-  if (is_null($field_storage_config_storage->load($data['id']))) {
-    $data['settings'] = $class::storageSettingsFromConfigData($data['settings']);
-    $field_storage_config_storage->create($data)->save();
-  }
-
-  /** @var \Drupal\Core\Config\Entity\ConfigEntityStorageInterface $field_config_storage */
-  $field_config_storage = $entity_type_manager->getStorage('field_config');
-  // Create field settings.
-  $data = $source->read('field.field.node.event.field_event_enable_enrollment_8916');
-  if (is_null($field_config_storage->load($data['id']))) {
-    $field_config_storage->create($data)->save();
-  }
-}
-
-/**
- * Update event display.
+ * Add translation compatibility for Event fields and views.
  */
-function social_event_update_8917() {
+function social_event_update_8919() {
   /** @var \Drupal\update_helper\Updater $updateHelper */
   $updateHelper = \Drupal::service('update_helper.updater');
 
   // Execute configuration update definitions with logging of success.
-  $updateHelper->executeUpdate('social_event', 'social_event_update_8917');
+  $updateHelper->executeUpdate('social_event', 'social_event_update_8919');
 
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
 
 /**
- * Set default configs for event.
+ * Update event values.
```

Comment:
We already have released an update hook with a bigger number: `social_event_update_10201`.
It means this update hook will never be run on sites that use OS 10.2.x and later.
I think we need to add this PR to the 10.3 milestone. It means you can use update hooks bigger than `social_event_update_10304`.